nemo/collections/asr/modules/conformer_encoder.py

```diff
@@ -252,6 +266,16 @@ def __init__(
         else:
             self.pre_encode = nn.Linear(feat_in, d_model)
 
+        # Reduction
+        if reduction and reduction_factor > 1:
+            self.reduction_subsampling = SubsamplingReductionModule(
+                reduction=reduction, d_model=d_model, reduction_factor=reduction_factor,
+            )
+            self.reduction_position = reduction_position
```

Comment:
Let's check that reduction_position has a valid value and raise an error otherwise.

Reply:
does an assert statement to ensure that the value remains between `[-1, len(self.layers)]` sound good?

Reply:
Yes, sounds good.